packages/server/test/unit/util/args_spec.js

```diff
@@ -595,6 +595,20 @@ describe('lib/util/args', () => {
         config: {},
       })
     })
+
+    it('moves testing-type specific config options', function () {
```

Comment:
Cool, looks good. Do we have a pre-existing pattern for if they pass invalid config, eg `--config={blablah=false}`, or we just ignore these? It looks like we will inherit the same behavior with the hoisted keys, just wanted to clarify.